packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx

```diff
@@ -50,21 +53,29 @@ export const HoldPane = (): JSX.Element => {
     };
   }, [startTime]);
 
+  const showSpinner = (): JSX.Element => {
+    return <Spinner label={strings.resumingCallButtonLabel} labelPosition={'right'} />;
+  };
+
   return (
     <Stack styles={paneStyles}>
       <Stack horizontal styles={holdPaneContentStyles}>
         <Text styles={holdPaneTimerStyles}>{elapsedTime}</Text>
         <Text styles={holdPaneLabelStyles}>{strings.holdScreenLabel}</Text>
         <PrimaryButton
-          text={strings.resumeCallButtonLabel}
-          ariaLabel={strings.resumeCallButtonAriaLabel}
+          text={!resumingCall ? strings.resumeCallButtonLabel : undefined}
+          ariaLabel={!resumingCall ? strings.resumeCallButtonAriaLabel : strings.resumingCallButtonAriaLabel}
           styles={resumeButtonStyles}
+          disabled={resumingCall}
           onClick={() => {
             /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
             holdButtonProps.onToggleHold();
```

Comment:
can this be awaited to catch an error, e.g. it would be great if this could be:
```
try {
  setResumingCall(true);
  await holdButtonProps.hold();
} finally {
  setResumingCall(false);
}
```

The key part there being the finally that resets the state so resume could be clicked again if it fails